app/assets/src/components/visualizations/dendrogram/Dendogram.js

```diff
@@ -345,6 +451,8 @@ export default class PhyloTree {
     let maxDistance = this.computeDistanceToRoot(this.root);
     this.createScale(-30, 0, this.minTreeSize.width, maxDistance);
     this.updateHighlights();
+    this.updateColors();
```

Comment:
This does not need to run for all update calls. Only when the tree is first drawn or rerooted.